### ISSUE TITLE
AVDCreateGPUAccelerator: Error loading GPU renderer

### ISSUE THREAD

**Issue by Sebvandenbrink** [ramboxapp/community-edition#979]
### Steps to reproduce

1. Start Rambox on Mac Pro

### Expected behavior

Starting Rambox won't give error (or open Terminal window).

### Actual behavior

On booting up on a Mac Pro (MacOS Sierra 10.12.5), starting Rambox opens up a Terminal window in which the error "AVDCreateGPUAccelerator: Error loading GPU renderer" is shown. Closing the Terminal window results in Rambox crashing.

From what I've gathered, this error is Mac Pro-specific, due to its Xeon CPU. See https://stackoverflow.com/questions/38466503/electron-error-avdcreategpuaccelerator-error-loading-gpu-renderer 
and
http://stackmirror.cn/page/s1go9ts4arzn

Despite the error, Rambox seems to function just fine. Strangely, though, after upgrading to 0.5.10 from the previous version, at first no Terminal opened. Yet, it returned on booting the next day.

### ENVIRONMENT

> Rambox: v0.5.10
> OS: MacOS Sierra 10.12.5
> Arch: x64 (Xeon CPU)

**Comment by saenzramiro** [comment]
I use it in Mac Sierra also and never displays that error.

Can you provide a screenshot please?

**Comment by Sebvandenbrink** [comment]
It seems Mac Pro specific due to what I've described above.

I've turned off hardware acceleration, that seems to have solved it. But I will leave it turned on for now so I can capture a screenshot when it does happen again. 

It only seems to trigger the Terminal when booting up, with Rambox starting on start-up. It is not reproducible when enabling/disabling hardware acceleration and quitting/restarting the program.
